sys.py/langs/05_Spanish.ini

```diff
@@ -16,14 +17,15 @@ Enter=Entrar
 Remove=Borrar
 Backspace=Borrar
 Done=Hecho
-Run=Lanzar
+Run=Abrir
 AddToPlayList=Añadir a lista
 Add to Playlist=Añadir a lista
 DownloadConfirm=Confirmar descarga
+Download Confirm=Confirmar descarga
```

Comment:
This string had an space between both words in the python script ("Download Confirm" instead of "DownloadConfirm"), so it was not getting the translation from this file.